smurfs/src/actions/smurfs.create.actions.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+import { baseURL, SERVICES } from '../api';
+
+const START_CREATING_SMURF = `START_CREATING_SMURF`;
```

Comment:
keep your naming conventions the same across action type groupings. aka change start to 'SMURG_CREATING_START'